### DISCUSSION TITLE
Caddy Reverse Proxy

### DISCUSSION THREAD

**Discussion by bkev** [logdyhq/logdy-core#38] [Q&A]
Is it possible to use Caddy as a reverse https proxy for Logdy?

My Logdy instance runs correctly on the web ui port over http, but when I try to configure Caddy to reverse proxy to that IP, the Logdy site loads but it says "Connection: Not Connected". 

I'm a little confused!

**Comment by PeterOsinski** [comment]
It's probably a problem with proxying websocket connection. Take a look how it was solved with Nginx: https://github.com/logdyhq/logdy-core/discussions/21

**Reply by bkev** [reply]
Thanks for the comment.

I've tried a few things but I can't seem to migrate the Nginx config to Caddy I'm afraid.

I do get the same error

https://IP:PORT/' was loaded over HTTPS, but attempted to connect to the insecure WebSocket endpoint 'ws://IP:PORT/ws?password=PASSWORD'. This request has been blocked; this endpoint must be available over WSS.

But I don't know how to do the sub filter in Caddy.

Thanks for the pointer though.

**Reply by PeterOsinski** [reply]
@bkev which Logdy version are you using? Recently there was a [change](https://github.com/logdyhq/logdy-ui/pull/6/files) that fixed the error above. The UI is trying to connect to websocket via unsecured protocol because it was hardcoded. That PR fixes it and it recently landed in the latest 0.13 release (homebrew not updated yet so please download a binary)

**Reply by bkev** [reply]
Thanks for helping.
I have upgrade to 0.13 but I get the same. I'm sure it's me, I don't know what websockets are and was just hoping a little bit of Caddy config would add https. 
It's not that important for me as only accessed internally.
I don't know how to convert the Nginx conf to Caddy so I'll just leave it for now.
